From f1f3e47cad42fcd3de898468a0af63d586b5ada3 Mon Sep 17 00:00:00 2001
From: Pietro Fanti <pietro.fanti@esa.int>
Date: Fri, 6 Jun 2025 11:55:48 +0200
Subject: [PATCH] fix: heyok v6 -> v7

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5aaeb04..b8385d6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -174,7 +174,7 @@ find_package(fmt REQUIRED CONFIG)
 target_link_libraries(cascade PUBLIC fmt::fmt)
 
 # heyoka.
-find_package(heyoka 6 REQUIRED CONFIG)
+find_package(heyoka REQUIRED CONFIG)
 target_link_libraries(cascade PUBLIC heyoka::heyoka)
 
 # spdlog.
@@ -183,7 +183,7 @@ target_link_libraries(cascade PRIVATE spdlog::spdlog)
 
 # Boost.
 # NOTE: need 1.73 for atomic_ref.
-# NOTE: it seems this must to be after find_package(heyoka 6 REQUIRED CONFIG) else 
+# NOTE: it seems this must to be after find_package(heyoka REQUIRED CONFIG) else 
 # cmake will complain of the missing target Boost::serialization
 find_package(Boost 1.73 REQUIRED CONFIG)
 target_link_libraries(cascade PUBLIC Boost::boost)
